tf/env/production/cluster.tf

```diff
@@ -60,6 +60,57 @@ resource "google_container_node_pool" "wbaas-3_highmem-16" {
   }
 }
 
+resource "google_container_node_pool" "wbaas-3_search-master-pool" {
+  cluster    = "wbaas-3"
+  name       = "search-master-pool"
+  node_count = 3
+  node_locations = [
+    "europe-west3-a",
+  ]
+  node_config {
+    disk_size_gb = 64
+    disk_type    = "pd-ssd"
+    machine_type = "n2-highmem-4"
+    metadata = {
+      "disable-legacy-endpoints" = "true"
+    }
+    oauth_scopes = [
+      "https://www.googleapis.com/auth/devstorage.read_only",
+      "https://www.googleapis.com/auth/logging.write",
+      "https://www.googleapis.com/auth/monitoring",
+      "https://www.googleapis.com/auth/service.management.readonly",
+      "https://www.googleapis.com/auth/servicecontrol",
+      "https://www.googleapis.com/auth/trace.append",
+    ]
+    preemptible     = false
+    service_account = "default"
+    shielded_instance_config {
+      enable_integrity_monitoring = true
+      enable_secure_boot          = false
+    }
+    labels = {
+      "wbaas/pool" = "search-master"
+    }
+    taint = [
+      {
+        key    = "wbaas/pool"
+        value  = "search-master"
+        effect = "NO_SCHEDULE"
+      }
+    ]
+    logging_variant = "MAX_THROUGHPUT"
```

Comment:
we might need to toggle this if really can't make stuff fit with fluentbit running here